prov/efa/src/rdm/rxr_pkt_type_base.c

```diff
@@ -121,8 +121,13 @@ int rxr_pkt_init_data_from_op_entry(struct rxr_ep *ep,
 		return 0;
 	}
 
-	ofi_iov_locate(op_entry->iov, op_entry->iov_count, tx_data_offset,
-		       &tx_iov_index, &tx_iov_offset);
+	ret = ofi_iov_locate(op_entry->iov, op_entry->iov_count, tx_data_offset,
+			     &tx_iov_index, &tx_iov_offset);
+	if (ret) {
```

Comment:
add `OFI_UNLIKELY`?

Reply:
Sure, there are some other places in efa call this function too, I can add it for them as well.

Reply:
Added